x-pack/plugins/security/public/views/management/roles_grid/components/permission_denied/permission_denied.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiEmptyPrompt, EuiPageContent } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React from 'react';
+
+export const PermissionDenied = () => (
+  <EuiPageContent horizontalPosition="center">
```

Comment:
Add this wrapper around the EuiPageContent to get it to center:

```jsx
<EuiFlexGroup gutterSize="none"></EuiFlexGroup>
```

Reply:
Should the `EuiPageContent` also be inside an `EuiFlexItem` since it's contained within an `EuiFlexGroup`?

Reply:
You'll be alright with `gutterSize="none"`. In order for `horizontalPosition="center"` to work on `EuiPageContent` it has to be a child of a flex parent. And so wrapping in the `EuiFlexGroup` is the easiest way to achieve that.